### DISCUSSION TITLE
Don't have permission message

### DISCUSSION THREAD

**Discussion by kenuna** [Zrips/CMI#7799] [Q&A]
How i can change this message? (like /gm if player don't have permission cmi.command.gamemode)
![image](https://user-images.githubusercontent.com/107900633/233838748-63e6d7b8-050a-47db-9fe0-5edc6f047a23.png)

**Comment by CloudeLecaw** [comment]
you find it in cmilib/translations in the used language file.

![image](https://github.com/Zrips/CMI/assets/12080017/d6f78fdd-d551-46b0-b0bd-b3acf54b56e5)

Also a duplicate of a issue quesion migh forgot to close issue one?
https://github.com/Zrips/CMI/issues/7800

**Comment by CloudeLecaw** [comment]
The Ticket was answered on 24th April. I think it can be closed now.
